app/screens/move-money-screen/move-money-screen.tsx

```diff
@@ -204,10 +203,9 @@ export const MoveMoneyScreenDataInjected: ScreenType = ({
     }
   }
 
-  const lastTransactions = find(data?.wallet, { id: "BTC" })?.transactions?.slice(
-    undefined,
-    3,
-  )
+  const lastTransactions = data?.wallet
```

Comment:
A `?` after wallet is needed or else the app crashes on this screen when logged out.